src/Servers/IIS/IIS/src/IISServerDefaults.cs

```diff
@@ -3,8 +3,14 @@
 
 namespace Microsoft.AspNetCore.Server.IIS
 {
+    /// <summary>
+    /// Defaults to configure IIS In-Process with.
+    /// </summary>
     public class IISServerDefaults
     {
+        /// <summary>
+        /// Default authentication scheme, which is "Windows".
+        /// </summary>
         public const string AuthenticationScheme = "Windows";
```

Comment:
const string. Uh oh.

Reply:
Why is this such an issue? Please explain; there are many places where we prefer const string due perf.

Reply:
Auth schemes must be const for use in the authorize attribute.

Reply:
Fair enough. @jkotalik since you asked - https://stackoverflow.com/a/755693. The issue is that the value is burned in to the callsite. So if you happen to change these constants, they will not be picked up until the app is recompiled.

Reply:
> Why is this such an issue? Please explain; there are many places where we prefer const string due perf.

Hmm, usually the opposite is the case: readonly field values are shared between assemblies when referened. Mean while const will embedded a new instance at compile time when referenced.

Checking shared values for equality can be short-circuited with a fast ReferenceEquals call.